deployment/factorycube-server/values.yaml

```diff
@@ -39,6 +39,18 @@ factoryinsight:
     requests:
        cpu: 200m      
 
+### grafana-proxy ###
+grafana-proxy:
+  enabled: true
+  image: unitedmanufacturinghub/grafana-proxy
+  replicas: 1
+  # factoryinsight api key
+  factoryinsight_key: "factoryinsight_key"
```

Comment:
Integrate keys via helm chart and using in values.yaml
```yaml
customers:
   test1: apikey 
```

See also factoryinsight deployment

Reply:
resolved in fc93d4430448482aa3d85b6b925d7d54a625fb02